### PR TITLE
update module

### PR DESCRIPTION
**Pull request by milestorme**
travis needs activation ( https://travis-ci.org/azerothcore/mod-item-level-up ),update module , cleanup and add  travis .

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
travis activated

File: conf/mod_levelitem.conf.dist [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+LevelItem.Announce = 1
+
+
+#
+#    Set item's max level
+#        Description: Max level the Item can be used up to
+#        Default:     Enabled = 80
+#                     
+
+LevelItem.MaxItemLevel = 80
```
Comment (BarbzYHOOL) [inline conf/mod_levelitem.conf.dist]: need one line break especially in config files (dunno how they are concatenated, so it's safer)

File: conf/conf.sh.dist [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+	
+#
+# CUSTOM
+#
+
+    DB_WORLD_CUSTOM_PATHS+=(
+        $MOD_LEVEL_ITEM_ROOT"/sql/world/"
+        )
```
Comment (BarbzYHOOL) [inline conf/conf.sh.dist]: need one line break to be safe

File: conf/mod_levelitem.conf.dist [hunk 3]
```diff
@@ -0,0 +1,28 @@
+[worldserver]
+###################################################################################################
+#	LEVEL UP ITEM
+###################################################################################################
+#
+# This module was downloaded from the AzerothCore module catalog. It has been modified and packaged
+# for use with the StygianCore repack (http://stygianthebest.github.io).
```
Comment (BarbzYHOOL) [inline conf/mod_levelitem.conf.dist]: could be removed imo (add in credits if needed)

File: conf/mod_levelitem.conf.dist (hunk 1 above)
Comment (milestorme) [reply conf/mod_levelitem.conf.dist]: made changes is this what you meant?

File: conf/conf.sh.dist (hunk 2 above)
Comment (milestorme) [reply conf/conf.sh.dist]: made changes is this what you meant?

**Comment by wizzymore** [comment]
Good job @milestorme 

Thank you for your contributions!

File: conf/conf.sh.dist (hunk 2 above)
Comment (BarbzYHOOL) [reply conf/conf.sh.dist]: ye

**Pull request merged**